src/containers/tutor-profile/video-presentation/VideoPresentation.jsx

```diff
@@ -2,17 +2,15 @@ import { useTranslation } from 'react-i18next'
 import Box from '@mui/material/Box'
 import Typography from '@mui/material/Typography'
 import VideoBox from '~/components/video-box/VideoBox'
-import videoImg from '~/assets/img/tutor-profile-page/presentationVideoImg.png'
 
-const VideoPresentation = () => {
+const VideoPresentation = (props) => {
```

Comment:
Maybe you convert this component to Typescript, since is not clear what should be in props